src/getMany/index.ts

```diff
@@ -0,0 +1,44 @@
+import firestore from '../adaptor'
+import { Collection } from '../collection'
+import { doc, Doc } from '../doc'
+import { ref, Ref } from '../ref'
+import { wrapData } from '../data'
+import undefined = require('firebase/empty-import')
```

Comment:
I feel like you should be able to use native `undefined`. If it's mandatory, please export it through [the adaptor](https://github.com/kossnocorp/typesaurus/tree/master/src/adaptor), so it would work both in Node.js and browser.

Reply:
Copied this from `get`, removed.